src/elements/Header/_Header.js

```diff
@@ -1,42 +1,98 @@
 import cx from 'classnames'
-import React, { createElement, Component, PropTypes } from 'react'
-
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+  iconPropRenderer,
+  imagePropRenderer,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+function _Header(props) {
+  const {
+    _sdClass, _headerElement,
+    left, center, right, justified, dividing, block, attached, floating,
+    icon, image, children, className,
+  } = props
+
+  const classes = cx(
```

Comment:
I think it's becoming apparent, my inferior knowledge of the symantic 
prop types needs a bit of work.  It might add the process, putting 
together a doc of definitive prop types for all components.  This might 
also help to visual how these can be simplified across all components 
and might the api as user friendly as possible?

I'd be happy to undertake this, as would help my knowledge if helping 
migrate these components?

> Levi Thomason <mailto:notifications@github.com>
> 24 June 2016 at 17:56
>
> In src/elements/Header/_Header.js 
> <https://github.com/TechnologyAdvice/stardust/pull/278#discussion_r68427374>:
>
> > import META from '../../utils/Meta'
> > +import {
> > + getUnhandledProps,
> > + iconPropRenderer,
> > + imagePropRenderer,
> > + useKeyOnly,
> > +} from '../../utils/propUtils'
> > +
> > +function _Header(props) {
> > + const {
> > + _sdClass, _headerElement,
> > + left, center, right, justified, dividing, block, attached, floating,
> > + icon, image, children, className,
> > + } = props
> > +
> > + const classes = cx(
>
> Let's add support for |disabled| 
> <http://semantic-ui.com/elements/header#disabled> and |color| 
> <http://semantic-ui.com/elements/header#colored>:
>
> <Header disabled />
>
> cx(
> useKeyOnly(disabled, 'disabled')
> )
> <Header color='red' />
>
> cx(
> color, // since only the value is used, there is no helper method
> )
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/TechnologyAdvice/stardust/pull/278/files/4317290b6fd2e4980cfd74e736287438f69d4c6c#r68427374>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAWHsy7SjrIp2M26spj0QJyzBLkFsA6Mks5qPAw0gaJpZM4I9saB>.
>

-- 
Sent from Postbox 
<https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach>